ui/src/views/PageNotFoundView.vue

```diff
@@ -5,13 +5,18 @@
     tag="section"
   >
     <v-row justify="center">
-      <v-col cols="auto">
-        page not found
+      <v-col id="error-view-content">
+        <h1>Page not found</h1>
+        <p>The page either doesn't exist or you don't have access to this page</p>
+        <v-img id="error-view-img" :src="mainPicture" contain />
         <!-- to do: check the log in state, redirect based on the user type -->
         <v-btn
+          id="error-view-btn"
           depressed
           large
           to="/"
+          :color="style.color.primary"
+          class="white--text"
```

Comment:
CSS class naming standard usually follows this convention: 
{component_name}-{modifier}

in this case, the class should be renamed as `text-white`

The `--` pattern we usually see is due to the BEM pattern, and with the BEM pattern, `--`  refers to a state change in the class. 

For example, 
`container` can have 2 states, `active`, and `inactive`
This way, when the container is active, we can add either of these two classes to the container: `container--active` `container--inactive`

As for the `__` naming pattern, they refer to children. 
For example, suppose we have 2 HTML tags residing inside a container, `picture` and  `text`.
<container><picture /> <text /> </container>
Then the class for the picture can be named as `container__picture` and for the text `container__text`.